### ISSUE TITLE
Python error

### ISSUE THREAD

**Issue by P4sm** [NabaviLab/VarSimLab#1]
Hi,

I got an error while running VarSimLab with the default settings. Here's the SIMULATION_IS_COMPLETE.txt log file:

[SIMULATION_IS_COMPLETE.txt](https://github.com/NabaviLab/VarSimLab/files/1379512/SIMULATION_IS_COMPLETE.txt)

**Comment by abdelrahmanhosny** [comment]
Make sure that the docker container is running from a folder that it has a write permission to. If you are using Windows, this should be some directory under C drive.

**Comment by P4sm** [comment]
Actually, I had to already edit the manifest.json file:
To
`{
    "reference": "hg19.fa",
    "targets": "hg19-targets.bed"
}
`
from

`{
    "reference": “hg19.fa",
    "targets": “hg19-targets.bed"
}
`
I changed the quotation marks. Otherwise the program gave me an error about the manifest file.

**Comment by P4sm** [comment]
I am using Ubuntu 16.04 LTS and I am running it as the superuser (root).

**Comment by abdelrahmanhosny** [comment]
Can you use the latest Docker image `nabavilab/varsimlab:latest` when you use Docker run. Make sure you have the latest image locally by using the command `docker pull nabavilab/varsimlab`

**Comment by P4sm** [comment]
And according to the log file the program completed some of the first steps, and it managed to write to the folder I am using.

**Comment by P4sm** [comment]
I just used this run.sh script:

`docker stop varsimlab`
`docker rm varsimlab`
`docker run -v $(pwd):/ref -p 8000:8000 --name varsimlab nabavilab/varsimlab`

and it loads VarSimLab v0.0.2

**Comment by abdelrahmanhosny** [comment]
What are the output files you get? What are their sizes?

**Comment by P4sm** [comment]
normal/
SIMULATION_IS_COMPLETE.txt
tumor/


normal/

normal_all_reads.fastq 4,9M

tumor/

tumor_all_subclones_reads.fastq 0
subclone_1/

tumor/subclone_1/

INDELs_1.txt 7,5M
INDELs_2.txt 7,5M
SNPs_1.txt 145M
SNPs_2.txt 145M

**Comment by abdelrahmanhosny** [comment]
Can you run the simulation again and let us know if you get the exact same error? The above file sizes (in megabytes) seem to be unexpected.

**Comment by P4sm** [comment]
I think the normal file is OK.

`cat normal_all_reads.fastq | wc -l`
`80000`

because I have just 10000 read pairs (20 000 reads)

**Comment by P4sm** [comment]
Yes, I got the same error and the same files again

**Comment by P4sm** [comment]
Before the error I had these files in tumor/subclone_1/

-rwxr-xr-x 1 root root 3157608038 loka  12 18:03 hg19.fa
-rw-r--r-- 1 root root    7868416 loka  12 18:20 hg19.fa_INDELs_S_0.0020_I_0.0001_C_2.00_1000_150000_1.txt
-rw-r--r-- 1 root root    7856128 loka  12 18:20 hg19.fa_INDELs_S_0.0020_I_0.0001_C_2.00_1000_150000_2.txt
-rw-r--r-- 1 root root  151148029 loka  12 18:18 hg19.fa_SNPs_S_0.0020_I_0.0001_C_2.00_1000_150000_1.txt
-rw-r--r-- 1 root root  151148029 loka  12 18:18 hg19.fa_SNPs_S_0.0020_I_0.0001_C_2.00_1000_150000_2.txt
-rw-r--r-- 1 root root 3157860352 loka  12 18:21 Indel_1.fa
-rw-r--r-- 1 root root 3157864448 loka  12 18:21 Indel_2.fa
-rw-r--r-- 1 root root    7852032 loka  12 16:45 INDELs_1.txt
-rw-r--r-- 1 root root    7860224 loka  12 16:45 INDELs_2.txt
-rw-r--r-- 1 root root 3157608088 loka  12 18:18 SNP_OUT_1.fa
-rw-r--r-- 1 root root 3157608088 loka  12 18:18 SNP_OUT_2.fa
-rw-r--r-- 1 root root  151148029 loka  12 16:43 SNPs_1.txt
-rw-r--r-- 1 root root  151148029 loka  12 16:43 SNPs_2.txt

And according to the log file the program is looking for /ref/my_sim/tumor/subclone_1/allele_1.fa
which doesn't exist

**Comment by P4sm** [comment]
I tried just the chromosome 1 of hg19 and no error this time. 
[SIMULATION_IS_COMPLETE.txt](https://github.com/NabaviLab/VarSimLab/files/1382126/SIMULATION_IS_COMPLETE.txt)

And I actually reinstalled docker, so now I can try with the whole hg19 reference.

**Comment by P4sm** [comment]
Yes, I got the same error again with the whole reference. Perhaps my computer ran out of memory? How much RAM should be enough for this?

**Comment by Eric4Wu** [comment]
I have same issue that chr1 did but whole chromosome did not simulate the reads, what does this message stands for :  "Wait! Wait! Simulating the variations in subclone 1 is complete. But it's still generating reads ..
Cleaning temporary files .." , it appears before allele_1.fa simulation step, and the next run the system can't find allele_1.fa then terminate the whole simulation. why whole chromosome simulation put this message before the allele.fa simulation ?

**Comment by abdelrahmanhosny** [comment]
@Eric4Wu Perhaps whole genome simulation suck up all the memory you have and get stuck at some point. I think it would be better to simulate on a single chromosome, not the whole genome.
